lua/apisix/plugins/heartbeat.lua

```diff
@@ -49,9 +49,13 @@ end
 
 local function report()
     -- ngx.sleep(3)
-    local etcd_version, err = core.etcd.server_version()
-    if not etcd_version then
-        core.log.error("failed to fetch etcd version: ", err)
+    local etcd_version = {}
+    if core.config.local_conf().apisix.config_center == "etcd" then
+        local err
+        etcd_version, err = core.etcd.server_version()
+        if not etcd_version then
```

Comment:
in the server side , we need to know whether it's error when call `core.etcd.server_version` or not use etcd as config center.

Reply:
good